java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -791,18 +796,102 @@ private static Object[] geospatialCommands(BaseTransaction<?> transaction) {
                 .geodist(geoKey1, "Palermo", "Catania", GeoUnit.KILOMETERS)
                 .geohash(geoKey1, new String[] {"Palermo", "Catania", "NonExisting"});
 
-        return new Object[] {
```

Comment:
Is that just moved, right?

Reply:
yes these becomes the initial `expectedResults`